commands/scripts/moderate_words.txt

```diff
@@ -1 +1,17 @@
-fuck motherfucker bitch
\ No newline at end of file
+English: 
```

Comment:
no need to mention the languages.... all words should be separated by a newline or space no colons or tab spaces

Reply:
Ok ok got it.